src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -53,19 +48,27 @@ At this point, the DevTool is not connected. To connect the DevTool to the appli
 adb shell input keyevent 82
 ```
 
-This will refresh the connection. 
+This will refresh the connection.
+
+### Exercise 1
+
+- Explore Components:
+  - Use the `Components` tab to explore the hierarchical structure of the React Native components in the application.
```

Comment:
Use backticks for code and quotes for everything else (so you can write “Components” instead).